apps/src/templates/projects/PersonalProjectsNameCell.jsx

```diff
@@ -12,35 +15,59 @@ class PersonalProjectsNameCell extends Component {
     isEditing: PropTypes.bool,
     updatedName: PropTypes.string,
     updateProjectName: PropTypes.func.isRequired,
+    isFrozen: PropTypes.bool,
+    showProjectInfoDialog: PropTypes.func,
   };
 
   onChangeName = e => {
     this.props.updateProjectName(this.props.projectId, e.target.value);
   };
 
+  showProjectInfo = () => {
+    console.log('showProjectInfo');
+    this.props.showProjectInfoDialog();
+  };
+
   render() {
-    const {projectId, projectType, projectName, updatedName, isEditing} =
-      this.props;
+    const {
+      projectId,
+      projectType,
+      projectName,
+      updatedName,
+      isEditing,
+      isFrozen,
+    } = this.props;
     const url = `/projects/${projectType}/${projectId}/edit`;
 
     return (
       <div>
         {!isEditing && (
-          <a
-            style={tableLayoutStyles.link}
-            href={url}
-            target="_blank"
-            rel="noopener noreferrer"
-            className="ui-projects-table-project-name"
-          >
-            {projectName}
-          </a>
+          <div>
+            <a
+              style={tableLayoutStyles.link}
+              href={url}
+              target="_blank"
+              rel="noopener noreferrer"
+              className="ui-projects-table-project-name"
+            >
+              {projectName}
+            </a>
+            {isFrozen && (
+              <button
+                type="button"
+                className={moduleStyles.infoButton}
+                onClick={this.showProjectInfo}
+              >
+                <FontAwesome icon="circle-exclamation" className="infoIcon" />
```

Comment:
nit: is it possible to use `FontAwesomeV6Icon` here instead of old component?

Reply:
Sure! Will do.